packages/main/test/pages/Popups.html

```diff
@@ -25,7 +25,7 @@
 
 	<link rel="stylesheet" type="text/css" href="styles/Popups.css">
 </head>
-<body class="popups1auto">
+<body class="popups1auto" onload="onload()">
```

Comment:
I would add `defer` to the script, move it on the bottom of the page or attach to the load event in the `popups.js` file rather than inlining this. Up to you, just a suggestion.